ppafm/cli/relaxed_scan.py

```diff
@@ -175,6 +175,9 @@ def main(argv=None):
             tip_positions_x, tip_positions_y, tip_positions_z, lvec_scan = common.prepareScanGrids()
             r_tips = np.array(np.meshgrid(tip_positions_x, tip_positions_y, tip_positions_z)).transpose(3, 1, 2, 0).copy()
             evals, evecs = core.stiffnessMatrix(r_tips.reshape((-1, 3)), pp_positions.reshape((-1, 3)), which=which, ddisp=common.params["ddisp"])
+            print("vib eigenval 1 min..max : ", np.min(evals[:, 0]), np.max(evals[:, 0]))
```

Comment:
It is not necessary, but I think it is useful to see some summary in the output imediately, before you start plotting images. In case you see some non-sense value (NaNs, all zero, super high, super low) then you know immedietely something went wrong. So yes, it is just for debugging, but I think it is debugging which user will often do.